### PR TITLE
SISRP-17104 Cache cleanup part 2: MyAcademics

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-17104

As with part 1, the commit messages tell the story.

### CODE REVIEW

**Comment by raydavis** [comment]
@johncrossman , @paulkerschen , could both of you look this over? Thanks!

**Comment by paulkerschen** [comment]
@raydavis, this all looks right and good to me. Passing over to @johncrossman.

**Comment by johncrossman** [comment]
@raydavis @paulkerschen  I'll review later this eve.

File: app/controllers/advising_student_controller.rb [hunk 1]
```diff
@@ -19,8 +19,7 @@ def profile
   def academics
     student_uid = student_uid_param
     render json: {
-      academics: MyAcademics::Merged.new(student_uid).get_feed,
-      examSchedule: MyAcademics::Exams.new(student_uid).merge
+      academics: MyAcademics::Merged.new(student_uid).get_feed
```
Comment (johncrossman) [inline app/controllers/advising_student_controller.rb]: Thanks, @raydavis, for this and the corresponding change in `advisingStudentController.js`

**Comment by johncrossman** [comment]
Looks good, @raydavis  Thanks for doing this delicate dance.

**Pull request merged**